manim/mobject/svg/style_utils.py

```diff
@@ -75,7 +75,7 @@ def parse_color_string(color_spec: str) -> str:
         if splits[0][-1] == "%":
             # if the last character of the first number is a percentage,
             # then interpret the number as a percentage
-            parsed_rgbs = [int(i[:-1]) / 100.0 for i in splits]
+            parsed_rgbs = [float(i[:-1]) / 100.0 for i in splits]
         else:
             parsed_rgbs = [int(i) / 255.0 for i in splits]
```

Comment:
@MrMallIronmaker maybe you want to do it here as well. Sorry, should have pointed that earlier.

Reply:
Are you taking about the non-percentage case? Not sure what line you're referring to.

Reply:
You are correct. Pls, ignore my previous suggestion. The "non-percentage" case will not be (should not be) float therefore casting by an int is okay.

Reply:
no worries :+1: